packages/web/src/components/artist-recommendations/ArtistRecommendations.tsx

```diff
@@ -139,8 +139,11 @@ export const ArtistRecommendations = forwardRef<
     )
   }, [dispatch, artistId])
 
-  const suggestedArtists = useSelector<CommonState, User[]>((state) =>
-    selectSuggestedFollowsUsers(state, { id: artistId })
+  const allRelatedArtists = useSelector<CommonState, User[]>((state) =>
+    selectRelatedArtistsUsers(state, { id: artistId })
```

Comment:
should we be using "suggestedArtists" since related artists i imagine is for the user-ilst?